sipservice/src/main/java/net/gotev/sipservice/SipAccount.java

```diff
@@ -86,7 +87,7 @@ public SipCall addOutgoingCall(final String numberToDial, boolean isVideo, boole
 
         // check if there's already an ongoing call
         int totalCalls = 0;
-        for (SipAccount _sipAccount: SipService.getActiveSipAccounts().values()) {
+        for (SipAccount _sipAccount : SipService.getActiveSipAccounts().values()) {
```

Comment:
Please check usage of this snippet.

Reply:
All changes will be accommodated in AND-3730 PR.